src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -156,7 +340,84 @@ public JTextField getNameInput(){
      *
      * @return extensionInput
      */
-    public JTextField getExtensionInput(){
-        return extensionInput;
+    public String getExtensionInput() {
+        return extensionInput.getSelectedItem().toString();
+    }
+
+    /**
+     * This method returns an array of String with information to search by size
+     *
+     * @return sizeData
+     */
+    public String[] getSizeInput(){
+        sizeData = new String[2];
```

Comment:
Array size should be 3.

Reply:
why should it be 3? the Array will contain only 3 values: {0,1,2}

Reply:
Updated, thanks!